src/desktop/bundled/QtColorWidgets/include/QtColorWidgets/color_wheel_private.hpp

```diff
@@ -20,6 +20,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
+#pragma once
```

Comment:
Thank you for sharing. 
I was under the impression that `#pragma once` was faster because the compiler didn't have to re-open the header file every time it saw it.
Compilers also do a "Multiple-Inclusion Optimization" to avoid re-opening header files on some patterns. Correct header guards and `#pragma once` should both trigger that optimization.
https://github.com/IncludeGuardian/multiple-inclusion-optimization-tests